server/Procfile

```diff
@@ -0,0 +1,5 @@
+web: gunicorn -b 0.0.0.0:$PORT -w 3 newsroom.web.app:app
+websocket: python -m newsroom.web.ws
+worker: celery -A newsroom.web.worker.celery -Q "${SUPERDESK_CELERY_PREFIX}newsroom" worker
+beat: celery -A newsroom.web.worker.celery beat --pid=
```

Comment:
not sure about the naming here, but `newsroom.web.worker` seems a bit strange, what about `newsroom.worker.celery` ?

Reply:
I made it only to show that these celery processes belong to `web` app (`NewsroomWebApp` is used). 
Actually everywhere I've tried to make an explicit separation between `web` and `news_api`.

If it's an issue I can move `worker.py` out of the `web` dir.